src/cts/src/TritonCTS.cpp

```diff
@@ -1221,6 +1237,18 @@ odb::dbITerm* TritonCTS::getFirstInput(odb::dbInst* inst) const
   return nullptr;
 }
 
+odb::dbITerm* TritonCTS::getFirstOutput(odb::dbInst* inst) const
```

Comment:
How is this different from dbInst::getFirstOutput() ?

Reply:
Didn't know about the API.   Replaced it with dbInst API.